tests/test.js

```diff
@@ -312,16 +320,18 @@ end`);
  * @param {import("../types").Highs} Module
  */
 function test_unbounded(Module) {
-  const sol = Module.solve(`Maximize a subject to a >= 1`);
+  const sol = Module.solve(`Maximize a
+  subject to
+  a >= 1
+  end`);
```

Comment:
The lp parser is not backwards compatible ?!

Reply:
Not quite, but to be fair, the previous example was arguably ill-formatted in the first place, due to the missing "End" line; https://github.com/ERGO-Code/HiGHS/issues/1449

Reply:
ok

Reply:
The requirement for "end" was added since the CPLEX `.lp` format expects it. Since Gurobi reads
> Maximize a subject to a >= 1
I'll look into removing the "end" requirement